soc/riscv/riscv-ite/it8xxx2/Kconfig.defconfig.series

```diff
@@ -0,0 +1,46 @@
+# Copyright (c) 2020 ITE Corporation. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+if (SOC_SERIES_RISCV32_IT8XXX2)
+
+config SOC_SERIES
+string
```

Comment:
Please fix the formatting here,  should be indentation like all other Kconfig files utilize.